### ISSUE TITLE
On upgrade to build 318 got mcrypt_get_iv_size(): Module initialization failed

### ISSUE THREAD

**Issue by scosist** [rainlab/builder-plugin#76]
Used composer to install:
`composer create-project october/october october-dev dev-develop`

I'm using `dev-develop` in this case. What happened to the stable branch (dev-stable on packagist)?

During an upgrade in the backend interface I got the following error:
```
mcrypt_get_iv_size(): Module initialization failed
/var/www/october-dev/vendor/laravel/framework/src/Illuminate/Encryption/Encrypter.php line 303
```

Recently upgraded from debian wheezy to debian jessie and thought maybe this might be related to the notices given in relation to the PHP upgrade and changes with mcrypt.

**Comment by scosist** [comment]
To clarify, mcrypt had already been installed and enabled:
```
apt-get install php5-mcrypt
php5enmod mcrypt
service apache2 restart
```

I tried `php artisan key:generate` first as recommended [here](http://stackoverflow.com/questions/23997121/lamp-laravel-errorexception-mcrypt-get-iv-size-module-initialization-failed) but that didn't seem to help.

More searching turned up recommendations to change the [cipher](http://laravel.io/forum/05-30-2014-mcrypt-error-after-new-pc-laravel-installation) in config/app.php to a [constant](http://stackoverflow.com/questions/14005669/php-mcrypt-keeps-throwing-module-initialization-failed-warning/22651645#22651645):
```
    'cipher' => MCRYPT_RIJNDAEL_256,
```
Once I did that I was able to proceed with the upgrade. I would submit a PR, but not sure if this or something else should be done to properly solve the issue.
